src/site/content/en/blog/form-fatigue-guide-user-input/index.md

```diff
@@ -0,0 +1,273 @@
+until the user starts typing in the element.
+
+<figure class="w-figure">
+  <img src="image-placeholder.png" alt="placeholder text for input field" style="max-width: 400px;">
+  <figcaption class="w-figcaption">
+    Fig. 2 — Placeholder text.
+  </figcaption>
+</figure>
+
+```html
+<input type="search" placeholder="Find anything home...">
+```
+
+<!-- TODO (robdodson): display code above or below figure?  -->
+<!-- TODO (meggin): display code above or below figure?  -->
+
+{% Aside 'caution' %}
+Don't use the `input` field `value` to set a placeholder as this can break autofill (described below).
+{% endAside %}
+
+{% Aside 'warning' %}
+Placeholders should not be used instead of labels, as they disappear as soon as users start to type, [can reduce accessibility](https://www.smashingmagazine.com/2018/06/placeholder-attribute/), and can't be automatically translated. If you choose not to display a label for a given input field, hide the `label` element off-screen using CSS so it is still read by screen readers.
+{% endAside %}
+
+If hints should remain in view while users type, consider:
+
++   Using a separate hint element above or below the `input` field:
+
+<figure class="w-figure">
+  <img src="image-hint-element.png" alt="field hint element displayed below input field" style="max-width: 400px;">
+  <figcaption class="w-figcaption">
+    Fig. 3 — hint element below input.
+  </figcaption>
+</figure>
+
+{% Aside 'aside' %}
+When using helper text in addition to a label, associate the text with the input using the `aria-describedby` attribute:
+{% endAside %}
+
+```html
+<label for="password">Password</label>
+<input type="password" id="password" aria-describedby="newpassHelper">
+<p id="newpassHelper" class="input-hint">Your password should be 8-20 characters 
+long and include 1 number and 1 special character.</p>
+```
+
++   Adding input masks:
+
+<figure class="w-figure">
+  <video autoplay loop muted playsinline>
+    <source src="video-input-masking.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Fig. 4 — input masking <a href="https://css-tricks.com/input-masking/">(source)</a>.
+  </figcaption>
+</figure>
+
+<!-- TODO (robdodson): is ok to use animation from   -->
+
+<div class="w-aside w-aside--note">
+  Ensure the contrast ratios of fields and labels are
+accessible to all users. Small text should have a contrast ratio of at least
+4.5:1 against its background. Large text (at 14-point bold, 18-point regular and
+up) should have a contrast ratio of at least 3:1 against its background.
+</div>
+
+
+## Choose the best input type
+
+Checkout forms require a broad range of input types. Users appreciate websites
+that automatically present number pads when typing credit cards. To display a
+matching keyboard for input fields, set the most appropriate `type` attribute,
+or consider creating a custom element for entering a quantity or date. 
+
+Common `type` attributes:
```

Comment:
I'd suggest cutting this, going straight into the example, and then linking out to the MDN input types list at the end in an aside: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types